english/alone/alone.md

```diff
@@ -58,4 +58,6 @@ Stranger's head starts bleeding and he run away from her room.
 
 Kristy screams again but no one bothers to check on her.
 
-And the screams continues to come again and again
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.